x/emissions/types/validations.go

```diff
@@ -212,9 +225,14 @@ func (bundle *WorkerDataBundle) Validate() error {
 	}
 
 	// Check signature from the bundle, throw if invalid!
-	src := make([]byte, 0)
-	src, _ = bundle.InferenceForecastsBundle.XXX_Marshal(src, true)
-	if !pubkey.VerifySignature(src, bundle.InferencesForecastsBundleSignature) {
+	buf := inferenceForecastsBundleBufferPool.Get().([]byte) // nolint:forcetypeassert
+	defer inferenceForecastsBundleBufferPool.Put(buf)        //nolint:staticcheck
+	buf = buf[:0]
```

Comment:
I wonder if we should make a wrapper type for `sync.Pool` that automatically zeroes the length of the buffers (and also does the type assertion).  That way nobody will forget to do this.